Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -841,25 +1142,32 @@ public class KotlinTokenizer: SwiftTokenizer {
         let keyTokens = tokenize(type.keyType, node: node)
         let valueTokens = tokenize(type.valueType, node: node)
         return
-            [type.newToken(.identifier, "Map", node), type.newToken(.startOfScope, "<", node)] +
+//            [type.newToken(.identifier, "Map", node), type.newToken(.startOfScope, "<", node)] +
+            // needs mutable here too, swift code things all maps are mutable
+            [type.newToken(.identifier, "MutableMap", node), type.newToken(.startOfScope, "<", node)] +
             keyTokens +
             [type.newToken(.delimiter, ", ", node)] +
             valueTokens +
             [type.newToken(.endOfScope, ">", node)]
     }
 
     open override func tokenize(_ type: FunctionType, node: ASTNode) -> [Token] {
-        return super.tokenize(type, node: node)
+        var tokens = super.tokenize(type, node: node)
             .replacing({ $0.value == "Void" && $0.kind == .identifier },
                        with: [type.newToken(.identifier, "Unit", node)])
+        while true { // remove _ in function type's parameters
+            if let i = tokens.index(where:{ $0.kind == .identifier && $0.value == "_"  }) {
+                tokens.remove(at:i)
+                tokens.remove(at:i)
+                continue
+            }
+            break
```

Comment:
Maybe this code could be improved by writting something like:

`tokens.replacing({  $0.kind == .identifier && $0.value == "_" }, with: [])` 

and cascading